extension/mark-unread.js

```diff
@@ -228,12 +229,35 @@ window.markUnread = (() => {
 		return $('#user-links a.name img').attr('alt').slice(1);
 	}
 
+	function unreadIndicatorIcon() {
+		const $notificationIndicator = $('.header-nav-link.notification-indicator');
+		const $notificationStatus = $($notificationIndicator).find('.mail-status');
+
+		const localNotifications = localStorage.unreadNotifications;
+		let hasLocalNotification = false;
+
+		if (JSON.parse(localNotifications).length > 0) {
+			hasLocalNotification = true;
+		}
+		let notificationsLabel = '';
+		if (hasLocalNotification) {
```

Comment:
No need for a `hasLocalNotification` variable. Just use `JSON.parse(localNotifications).length > 0` directly as the condition of this `if` statement